pinot-core/src/test/java/org/apache/pinot/core/data/function/InbuiltFunctionEvaluatorTest.java

```diff
@@ -26,6 +26,7 @@
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
+import static org.testng.AssertJUnit.fail;
```

Comment:
(nit)
```suggestion
import static org.testng.Assert.fail;
```

Reply:
sounds good although we should really do these automatically. not a great use of any committer's time :-)